src/components/admin/edit-modal.tsx

```
@@ -657,55 +692,73 @@ export function EditModal({
                 {/* アバター名 */}
                 <div>
                   <label className="block text-gray-700 text-sm font-medium mb-1">
-                    アバター名
+                    {isWorldEntry ? '名称' : 'アバター名'}
                   </label>
```

Biome (2.4.4)

[error] A form label must be associated with an input.

(lint/a11y/noLabelWithoutControl)